### DISCUSSION TITLE
Lithuanian language

### DISCUSSION THREAD

**Discussion by MrVinyla** [Ylianst/MeshCentral#4569] [Q&A]
Hi, is there any way to add Lithuanian language option in the list. I will be able to translate with MeshCentral Translator when there is an "lt-Lithuanian" option. Thank you.
![lt](https://user-images.githubusercontent.com/55458509/191365614-0a713227-87a8-4c99-ae44-49ac08d4e466.png)

**Comment by Ylianst** [comment]
Hi. Yes. I am on vacation until October 18th. Once I get back I can add this language.

**Reply by MrVinyla** [reply]
Hi Ylian
I just want to remind you that I am waiting for you to add the Lithuanian language, as you promised. Thank you.
